modules/organismFactory.js

```diff
@@ -0,0 +1,59 @@
+function OrganismFactory(){}
```

Comment:
stick to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Introduction_to_Object-Oriented_JavaScript#Custom_objects